Makefile

```
@@ -40,6 +40,24 @@ $(release_drivers): sgp-common/sgp_git_version.c
 	cd release && zip -r "$${pkgname}.zip" "$${pkgname}" && cd - && \
 	ln -sf $${pkgname} $@
 
+release/sgp40: sgp-common/sgp_git_version.c
```

on second thought, I think it'd be wiser to replace it with the generic `prepare` target.. (also above)
```suggestion
release/sgp40: prepare
```

---

I change it to let all release targets just depend on prepare, so fixed.